.github/workflows/presubmit.yml

```diff
@@ -27,11 +28,11 @@ jobs:
           paths_ignore: '["**/README.md"]'
           do_not_skip: '["pull_request"]'
 
-  run_tests:
+  thapi_dependencies:
```

Comment:
can we do? `install_thapi_dependencies` and then `install_thapi`? I'm older and older, my brain is working less and less, I need explicit name more and more :'0